packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VXZoom/VXZoom.js

```diff
@@ -115,16 +187,21 @@ class VXZoom extends PureComponent {
               {...this.props}
             >
               <ZoomEventLayer
-                onWheel={(event) => this.onWheel(event)}
+                focusable
+                height={height}
+                left={left}
+                onDoubleClick={this.onDoubleClick}
+                onKeyDown={this.onKeyDown}
```

Comment:
I'm not sure why this isn't catching the pan/zoom keys when the 'Zoom enabled' button has focus.